security/keyutils-tests.py.data/keyutils.yaml

```diff
@@ -0,0 +1,6 @@
+run_type: !mux
+    upstream:
+        type: 'upstream'
+    distro:
+        type: 'distro'
+url: "https://git.kernel.org/pub/scm/linux/kernel/git/dhowells/keyutils.git"
```

Comment:
Ok with the above yaml, bothe distro and upstream cases gets executed every time

Reply:
Looks good, but the other way would be of url is given assume as upstream run else go for distro run   .. So in that case the type param can be eliminated

Reply:
from code, url is required for `upstream` type only.
so we can remove the default url and hard_code it in yaml.
@nasastry